app/static/src/app/apiService.ts

```diff
@@ -31,7 +31,9 @@ export interface API<S, E> {
     get<T>(url: string): Promise<void | ResponseWithType<T>>
 }
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Bafflingly, had to this to keep eslint happy, no idea why it thinks these types are unused, as they clearly are